shared/actions/login/index.js

```diff
@@ -475,6 +479,29 @@ function makeKex2IncomingMap (dispatch, getState) : incomingCallMapType {
       generateQRCode(dispatch, getState)
       dispatch(askForCodePage(phrase => { response.result({phrase, secret: null}) }, response))
     },
+    'keybase.1.provisionUi.chooseDeviceType': ({sessionID, kind}, response) => {
```

Comment:
We weren't handling this RPC before -- the ExistingDevice component wasn't hooked up to anything.